docs/source/llms/langchain/autologging.rst

```diff
@@ -0,0 +1,257 @@
+MLflow Langchain Autologging
+============================
+
+MLflow LangChain flavor supports autologging, a powerful feature that allows you to log crucial details about the LangChain model and execution without the need for explicit logging statements. MLflow LangChain autologging covers various aspects of the model, including traces, models, signature, and and more.
+
+.. attention::
+
+    MLflow LangChain Autologging feature is largely renewed in ``MLflow 2.14.0``. If you are using the earlier version of MLflow, please refer to legacy documentation `here <#documentation-for-old-versions>`_ for the old version of the feature.
+
+.. contents:: Table of Contents
+  :local:
+  :depth: 1
+
+Quickstart
+----------
+
+To enable autologging for LangChain models, call :py:func:`mlflow.langchain.autolog()` at the beginning of your script or notebook. This will automatically log the traces by default, and other artifacts such as models, datasets, and model signatures if you enable them. For more information about the configuration, please refer to the `Configure Autologging <#configure-autologging>`_ section.
+
+.. note::
```

Comment:
Is there a minimum version for trace integration support, or is this it?

Reply:
Great callout! Actually, now that we don't rely on the legacy callback in LangChain repo and other autolog artifacts are just subset of model logging, autolog does not have particular version requirement more than general LangChain-MLflow integration. Discussed with Serena offline and we just remove this extra warning.